files/zh-tw/web/html/element/article/index.md

```diff
@@ -0,0 +1,153 @@
+    <tr>
+      <th scope="row">標籤省略</th>
+      <td>{{no_tag_omission}}</td>
+    </tr>
+    <tr>
+      <th scope="row">允許的父元素</th>
+      <td>
+        任何接受
+        <a href="/zh-TW/docs/Web/HTML/Content_categories#flow_content"
+          >流動內容</a
+        > 的元素。請注意，<code>&#x3C;article></code> 元素不得是
+        {{HTMLElement("address")}} 元素的子孫。
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">隱含 ARIA 角色</th>
+      <td>
+        <code
+          ><a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/Article_Role"
+            >article</a
+          ></code
+        >
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">允許的 ARIA 角色</th>
+      <td>
+        <a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/application_role"><code>application</code></a>、<a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/document_role"><code>document</code></a>、
+        <a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/feed_role"><code>feed</code></a>、<a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/main_role"><code>main</code></a>、
+        <a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/none_role"><code>none</code></a>、<a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/presentation_role"><code>presentation</code></a>、
+        <a href="/zh-TW/docs/Web/Accessibility/ARIA/Roles/region_role"><code>region</code></a>
+      </td>
+    </tr>
+    <tr>
+      <th scope="row">DOM 介面</th>
+      <td>{{domxref("HTMLElement")}}</td>
+    </tr>
+  </tbody>
+</table>
+
+## 屬性
+
+此元素僅包含[全域屬性](/zh-TW/docs/Web/HTML/Global_attributes)。
+
+## 使用注意事項
+
+- 每個 `<article>` 應該被識別，通常通過在 `<article>` 元素的子元素中包含標題（[`<h1>` - `<h6>`](/zh-TW/docs/Web/HTML/Element/Heading_Elements) 元素）。
+- 當嵌套 `<article>` 元素時，內部元素代表與外部元素相關的文章。例如，部落格文章的評論可以是嵌套在表示部落格文章的 `<article>` 中的 `<article>` 元素。
+- `<article>` 元素的作者信息可以通過 {{HTMLElement("address")}} 元素提供，但不適用於嵌套的 `<article>` 元素。
+- `<article>` 元素的發佈日期和時間可以使用 {{HTMLElement("time")}} 元素的 [`datetime`](/zh-TW/docs/Web/HTML/Element/time#datetime) 屬性來描述。
+
+## 範例
+
+```html
+<article class="film_review">
+  <h2>Jurassic Park</h2>
+  <section class="main_review">
+    <h3>Review</h3>
+    <p>Dinos were great!</p>
+  </section>
+  <section class="user_reviews">
+    <h3>User reviews</h3>
+    <article class="user_review">
+      <h4>Too scary!</h4>
+      <p>Way too scary for me.</p>
+      <footer>
+        <p>
+          Posted on
+          <time datetime="2015-05-16 19:00">May 16</time>
+          by Lisa.
+        </p>
+      </footer>
+    </article>
+    <article class="user_review">
+      <h4>Love the dinos!</h4>
+      <p>I agree, dinos are my favorite.</p>
+      <footer>
+        <p>
+          Posted on
+          <time datetime="2015-05-17 19:00">May 17</time>
+          by Tom.
+        </p>
+      </footer>
+    </article>
+  </section>
+  <footer>
+    <p>
+      Posted on
+      <time datetime="2015-05-15 19:00">May 15</time>
+      by Staff.
+    </p>
+  </footer>
+</article>
+```
+
+## 結果
+
+{{EmbedLiveSample('Examples','','570')}}

```

Comment:
[translate the ID](https://github.com/mdn/translated-content/blob/main/docs/zh-tw/translation-guide.md#%E7%BF%BB%E8%AD%AF%E6%A8%99%E9%A1%8C-id)

```suggestion
{{EmbedLiveSample('範例','','570')}}
```